src/net/sourceforge/kolmafia/objectpool/AdventurePool.java

```diff
@@ -59,10 +59,10 @@ public class AdventurePool {
   public static final int FILTHWORM_FEEDING_CHAMBER = 128;
   public static final int FILTHWORM_GUARDS_CHAMBER = 129;
   public static final int FILTHWORM_QUEENS_CHAMBER = 130;
-  public static final int WARTIME_FRAT_HOUSE_DISGUISED = 131;
+  public static final int WARTIME_FRAT_HOUSE_DISGUISED = 134;
   public static final int FRAT_UNIFORM_BATTLEFIELD = 132;
-  public static final int WARTIME_HIPPY_CAMP = 133;
-  public static final int WARTIME_HIPPY_CAMP_DISGUISED = 134;
+  public static final int WARTIME_HIPPY_CAMP = 131;
+  public static final int WARTIME_HIPPY_CAMP_DISGUISED = 133;
```

Comment:
These are out of order and incorrect.

131 - wartime hippy camp disguised
(132 - frat uniform battlefield)
133 - wartime hippy camp
134 - wartime frat house disguised
135 - wartime frat house

Reply:
I swear I checked these... 

Yeah. Am I misreading/misunderstanding?

<img width="345" alt="Screenshot 2022-08-10 at 17 35 40" src="https://user-images.githubusercontent.com/179758/183964869-c2de9370-ff22-4831-b9b8-fe07028d0c05.png">

Reply:
Oh I see I'm getting confused around verge of war and wartime. Eugh this is confusing.

Reply:
Yeah. "Wartime" is when the hippy camp and frat house are coinmaster shops, not adventuring locations.